digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/DamagedByThawFiller.scala

```diff
@@ -32,18 +30,18 @@ class DamagedByThawFiller extends AssetFiller {
     }
 
     def toCurrentYear(period: DatePeriodValue): DatePeriodValue = {
-      val endDate = stringToDate(period.endDate)
-      val startDate = stringToDate(period.startDate)
+      val endDate = DateParser.stringToDate(period.endDate, DateParser.DatePropertyFormat)
```

Comment:
The stringToDate and dateToString methods used on the tests should be the same that is used here. The methods on this class can then also be removed.

Reply:
Done